doc/application/index.rst

```diff
@@ -248,14 +248,10 @@ should know about.
    * As :ref:`env_vars`.
    * As a ``set(<VARIABLE> <VALUE>)`` statement in your :file:`CMakeLists.txt`
 
-* :makevar:`ZEPHYR_BASE`: Sets the path to the directory containing Zephyr,
-  which is needed by the build system's boilerplate file.  This is an
-  environment variable set by the :file:`zephyr-env.sh` script on Linux/macOS
-  or :file:`zephyr-env.cmd` on Windows, as you learned when getting started
-  with Zephyr in :ref:`getting_started_run_sample`. You can also set
-  :makevar:`ZEPHYR_BASE` explicitly, but then you won't get the other features
-  provided by those scripts.
-
+* :makevar:`ZEPHYR_BASE`: Zephyr base variable used by the build system.
+   ``find_package(Zephyr)`` will automatically set this as a cached CMake
```

Comment:
It looks like there is one space too many at the beginning of this line and the next two. Since .rst is sensitive to whitespace, might be a good idea to remove them if so.

Reply:
fixed